GTBitsOfGood/national-npp
# Implement endpoint to change application stage

## What is this?
Implement an endpoint to change application stages. Includes MongoDB action for National Admins to modify projects.

## Setup

## Steps to Test
Create a Postman request to `/api/applications/[id]/natlAdmin` for some application ID from MongoDB. It should be PATCH, and write the body as raw JSON in the format of `{ stage: "<stage>"}`. You need to authenticate online in the web app and copy the cookies into Postman in order for the request to be authorized.

## Associated Issues
#86
